### ISSUE TITLE
Downloaded ISO no boot

### ISSUE THREAD

**Issue by symgryph** [sonertari/UTMFW#8]
I was trying to boot the ISO, no luck. Do I have to be in 'legacy' mode? UEFI? I did 'dual' but even my iso file seems corrupted (I got the one from the google drive). To build do I need an openbsd vm?

thomas

**Comment by symgryph** [comment]
Also trying to 'build' I could work on docs for build in markdown if you want.

**Comment by symgryph** [comment]
Looks like a lot of manual reading. I would be willing to document whats going on to make more accessible, but following openbsd docs less than easy. I guess the iso you link to isn't bootable, I expanded, but looks like I need to somehow create the 'bootable' iso? great looking project if I can ever figure out how to install it.

**Comment by sonertari** [comment]
Did you verify the iso file using its SHA256 checksum you can find on the README? If it verifies, the iso file should be fine.

I use VMware Player to test the system before release. It should work on other VM software and bare metal hardware, but perhaps you can try with VMware Player too. Make sure you configure 2 NICs, but other than that you shouldn't need to change any other configuration of the VMware BIOS (legacy, UEFI or otherwise).

IMO, the OpenBSD installation is the easiest I have ever seen in the past 25 years (I can install UTMFW in under 5 minutes). So, thanks for the offer but I think the documentation is mostly fine, you just need to get used to it (but feel free to report any issues in the InstallationGuide.txt in the UTMFW iso).

In short, the iso file should work out of the box. I think you are making a (or a couple of) simple mistake(s). I hope you can successfully install and try UTMFW. Please report back your further experiences.

**Comment by buraksn** [comment]
Hi, , have same problem.
@sonertari  your image does not working on usb memsticks . So i have tried to make memstick image from your iso image.
It booted and sets installed cuccessfully but my memstick image have a problem until installing your packages it hangs on installing packages stage installer can not finds packages directory  but it exist in main image directory. I gave directory path but no luck  Could you help me ? Any help would be appreciated.

## my work to convert iso image to usb memstick image

//openbsd side 1
vnconfig vnd0 utmfw68_20210211_amd64.iso
mount -t cd9660 /dev/vnd0c /mnt

//freebsd side 1
wget https://ftp.openbsd.org/pub/OpenBSD/6.8/amd64/install68.img
sudo dd bs=1M if=install68.img of=/dev/sdd

/ increasing part area to put "packages"
gpart show da1
gpart resize -i 4 -a 4k -s 10G da1
growfs /dev/da1s4

mount /dev/da1s4 /mnt
rm -r /mnt/6.8/
rm -r /mnt/etc/

// openbsd side 2
scp -r 6.8 root@192.168.2.10:/mnt/.
scp -r 6.8 root@192.168.2.10:/mnt/.
scp -r packages root@192.168.2.10:/mnt/.


//freebsd side 2
umount /mnt

**Comment by sonertari** [comment]
Yes, as I mention in README, the iso file is a CD image, so you are expected to burn it on a CD or use it directly to install on a Virtual Machine.

However, the following links may help you convert the iso file to a USB image (note that I have not tried these myself, but they seem simple enough)l: https://www.reddit.com/r/openbsd/comments/9xitpz/need_help_with_usb_install/ and http://liveusb-openbsd.sourceforge.net/. Please report back your experiences.

Perhaps I should also distribute a USB image in future releases.

**Comment by sonertari** [comment]
I can confirm that the instructions on [this OpenBSD journal article](https://undeadly.org/cgi?action=article;sid=20140225072408) work fine. I was able to convert the iso file for UTMFW 6.8 into a bootable USB image, write it to a USB stick, and install UTMFW as I would with a CD drive.

The only catch is that the installation cannot find the install sets. At that point you should tell it to use the `disk` option, that the disk partition is not mounted yet, and point it to the USB drive with the correct path to the install sets (the default path it offers is the same path on the UTMFW image too, so you just hit Enter at that point). That's it.

**Comment by buraksn** [comment]
Thanks, that works! i have solved the problem and not i have successfully made usb memstick image according to instructions from link on your post.
for beginners > do not forget answer "fdisk -i vnd1" to "y"

installing sets
![signal-2021-05-14-112550_006](https://user-images.githubusercontent.com/34309693/118243844-90fd7580-b4a7-11eb-9d20-e5c3cdef3e27.jpeg)

installing packages
![signal-2021-05-14-112550_002](https://user-images.githubusercontent.com/34309693/118243994-bf7b5080-b4a7-11eb-8cec-891ddc7024f5.jpeg)

UTMFW boot on my laptop
 
![signal-2021-05-14-112524](https://user-images.githubusercontent.com/34309693/118244047-c99d4f00-b4a7-11eb-8b0a-e381cfe8a470.jpeg)

**Comment by buraksn** [comment]
Here is downloadable usb memstick image [utmfw68_20210211_amd64.img](https://drive.google.com/file/d/1_YFFNAm1iI1C4LF2SyM3NxEFEjDPDKtM/view?usp=sharing)

SHA256 (utmfw68_20210211_amd64.img) = b9ce84d94f96dfc057e6792536d9510754461f5e54e10364afa5b1997387598a
